exec.go

```diff
@@ -48,12 +51,27 @@ func (exec *EXEC) Exports() modules.Exports {
 // Command is a wrapper for Go exec.Command
 func (*EXEC) Command(name string, args []string, option CommandOptions) string {
 	cmd := exec.Command(name, args...)
+
+	var outb, errb bytes.Buffer
+	cmd.Stdout = &outb
+	cmd.Stderr = &errb
+
 	if option.Dir != "" {
 		cmd.Dir = option.Dir
 	}
-	out, err := cmd.Output()
+
+	if len(option.Env) > 0 {
+		cmd.Env = append(cmd.Environ(), option.Env...)
+	}
+	err := cmd.Run()
 	if err != nil {
-		log.Fatal(err.Error() + " on command: " + name + " " + strings.Join(args, " "))
+		// Keep default behaviour backwards compatible
+		if option.FatalError == nil || *option.FatalError {
+			log.Fatal(err.Error() + " on command: " + name + " " + strings.Join(args, " "))
+		} else {
+			log.Print(err.Error() + " on command: " + name + " " + strings.Join(args, " "))
```

Comment:
I would really like to also return the error code, so that checks could be done on that value, but I also wouldn't like to change backwards compatibility.

I could add another method that would return a struct, something like:

```
{
  statusCode: 1
  stdError: ""
  stdOut: "" 
}
```

Reply:
Also, some statistics on the time it took to run the command would be nice...